src/content/app/genome-browser/services/browserStorageService.ts

```diff
@@ -99,6 +104,18 @@ export class BrowserStorageService {
     this.storageService.update(StorageKeys.TRACK_PANELS, trackPanels);
   }
 
+  public getBrowserBookmarks(): {
+    [genomeId: string]: Partial<BrowserBookmarksState>;
+  } {
+    return this.storageService.get(StorageKeys.BOOKMARKS) || {};
+  }
+
+  public updateBrowserBookmarks(browserBookmarks: {
+    [genomeId: string]: Partial<BrowserBookmarksStateForGenome> | undefined;
+  }): void {
+    this.storageService.update(StorageKeys.BOOKMARKS, browserBookmarks);
+  }
```

Comment:
I am confused. What is the purpose of these two methods compared to browserBookmarksStorageService?

Reply:
I was not going through all of the functions in these files but just renaming them, hence some legacy code remains. My bad.

We had a `TrackPanelStorageService` which was renamed to `BrowserBookmarksStorageService`: https://github.com/Ensembl/ensembl-client/blob/dev/src/content/app/genome-browser/components/track-panel/services/track-panel-storage-service.ts

When going through the code now, found out these two are interchangeably used to update the web storage which isn't right. Think we can consolidate the related code from both these files into `BrowserBookmarksStorageService` instead?

Reply:
As we discussed during code review, everything related bookmarks should be moved into `BrowsersBookmarksStorageService`.

Reply:
I agree to that as well @imransl . It would be good to move the code related to bookmarks into `BrowserBookmarksStorageService`. 

Regarding creating a new `trackPanelStorageService`, I don't think we need it at this point or in this PR.

Reply:
I agree to that as well @imransl . It would be good to move the code related to bookmarks into `BrowserBookmarksStorageService`. 

Regarding creating a new `trackPanelStorageService`, I don't think we need it at this point or in this PR.